### PR TITLE
Bintray download links in changelog

### PR DESCRIPTION
**Pull request by timbeiko**
Adds the bintray links and checksums to the bottom of each release's notes since 1.5.0 and points to the bintray directory for earlier version. Updated to reflect the changes done via the Github UI on our [Releases](https://github.com/hyperledger/besu/releases/) page.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,10 @@
 - [Privacy users with private transactions created using v1.3.4 or earlier](KNOWN_ISSUES.md#privacy-users-with-private-transactions-created-using-v134-or-earlier)
 - [Changes not saved to database correctly causing inconsistent private states](KNOWN_ISSUES.md#Changes-not-saved-to-database-correctly-causing-inconsistent-private-states)
 
+### Download link
+https://dl.bintray.com/hyperledger-org/besu-repo/besu-1.5.4.zip
```
Comment (shemnon) [inline CHANGELOG.md:25]:
```suggestion
https://dl.bintray.com/hyperledger-org/besu-repo/besu-1.5.4.zip

```

without the whtespace the sha values renders on the same line.

File: CHANGELOG.md [hunk 2]
```diff
@@ -54,6 +58,10 @@ To enhance control over permissions on the privacy group management contract:
 
 The onchain privacy support in the current version of the web3js-eea library (v0.9) will not be compatible with Besu v1.5.3.  We are actively working on an upgrade to webj3-eea that will support these changes.   
 
+### Download link
+https://dl.bintray.com/hyperledger-org/besu-repo/besu-1.5.3.zip
```
Comment (shemnon) [inline CHANGELOG.md:62]:
```suggestion
https://dl.bintray.com/hyperledger-org/besu-repo/besu-1.5.3.zip

```

File: CHANGELOG.md [hunk 3]
```diff
@@ -91,6 +99,10 @@ The intent is that the major Java VM version or Java VM type shipped with the de
 - [David Mechler](https://github.com/hyperledger/besu/commits?author=davemec) has been added as a [new maintainer](https://github.com/hyperledger/besu/pull/1267).
 - [Edward Evans](https://github.com/hyperledger/besu/commits?author=EdJoJob) voluntarily moved to [emeritus status](https://github.com/hyperledger/besu/pull/1270).
 
+### Download link
+https://dl.bintray.com/hyperledger-org/besu-repo/besu-1.5.2.zip
```
Comment (shemnon) [inline CHANGELOG.md:103]:
```suggestion
https://dl.bintray.com/hyperledger-org/besu-repo/besu-1.5.2.zip

```

File: CHANGELOG.md [hunk 4]
```diff
@@ -127,6 +139,10 @@ Known issues are open issues categorized as [Very High or High impact](https://w
 - [Permissioning issues on Kubernetes](KNOWN_ISSUES.md#Kubernetes-permissioning-uses-Service-IPs-rather-than-pod-IPs-which-can-fail)
 - [Restarts caused by insufficient memory can cause inconsistent private state](KNOWN_ISSUES.md#Restart-caused-by-insufficient-memory-can-cause-inconsistent-private-state)
 
+### Download link
+https://dl.bintray.com/hyperledger-org/besu-repo/besu-1.5.1.zip
```
Comment (shemnon) [inline CHANGELOG.md:143]:
```suggestion
https://dl.bintray.com/hyperledger-org/besu-repo/besu-1.5.1.zip

```

File: CHANGELOG.md [hunk 5]
```diff
@@ -215,12 +229,18 @@ Workaround - Do not use permissioning on K8S.
...
 #### Previously identified known issues
-
+ 
 - [Scope of logs query causing Besu to hang](KNOWN_ISSUES.md#scope-of-logs-query-causing-besu-to-hang)
 - [Eth/65 loses peers](KNOWN_ISSUES.md#eth65-loses-peers)
 - [Fast sync when running Besu on cloud providers](KNOWN_ISSUES.md#fast-sync-when-running-besu-on-cloud-providers)
 - [Privacy users with private transactions created using v1.3.4 or earlier](KNOWN_ISSUES.md#privacy-users-with-private-transactions-created-using-v134-or-earlier)
 
+### Download link
+https://dl.bintray.com/hyperledger-org/besu-repo/besu-1.5.0.zip
```
Comment (shemnon) [inline CHANGELOG.md:239]:
```suggestion
https://dl.bintray.com/hyperledger-org/besu-repo/besu-1.5.0.zip

```

**Review by shemnon — APPROVED** [review]

**Comment by timbeiko** [comment]
Thanks for the catch, @shemnon 😄

**Pull request merged**